docs/basic-features/script.md

```diff
@@ -19,10 +19,11 @@ description: Next.js helps you optimize loading third-party scripts with the bui
 <details>
   <summary><b>Version History</b></summary>
 
-| Version   | Changes                   |
-| --------- | ------------------------- |
-| `v12.2.4` | `onReady` prop added.     |
-| `v11.0.0` | `next/script` introduced. |
+| Version   | Changes                                               |
+| --------- | ----------------------------------------------------- |
+| `v12.2.4` | `onReady` prop added.                                 |
+| `v12.2.2` | Allow `next/script` to be placed in `_document` body. |
```

Comment:
```suggestion
| `v12.1.6` | Allow `next/script` with `beforeInteractive` to be placed in `_document`. |
```
The related PR for this was https://github.com/vercel/next.js/pull/31936 which was first available in `v12.1.6` it looks like

Reply:
This was changed in [release 12.2.2](https://github.com/vercel/next.js/releases/tag/v12.2.0) right? [This](https://github.com/vercel/next.js/pull/37894/) is the PR.
I found this within the 12.2.2 changelogs:
```
[Script] Allow next/script to be placed in _document body: https://github.com/vercel/next.js/pull/37894
```

Reply:
Marking this as `12.2.2` sounds good as the above bug fix was crucial before it was fully working in `_document`.